bin/oref0-setup.sh

```diff
@@ -221,6 +221,14 @@ if openaps vendor add --path . mmeowlink.vendors.mmeowlink 2>&1 | grep "No modul
     echo Installing latest mmeowlink
     sudo pip install -U mmeowlink || die "Couldn't install mmeowlink"
 fi
+# Install Bluez for BT Tethering
```

Comment:
Perhaps condition it on https://github.com/openaps/oref0/blob/dev/bin/oref0-setup.sh#L75 ? Although that leaves a bit of a bootstrapping issue if you don't have bluez yet and can't look up your phone's MAC...